scripts/linters/general_purpose_linter.py

```diff
@@ -122,6 +122,8 @@
             # same issue. The following was placed due to a necessary sleep as
             # a temporary measure to keep the embedding tests from failing.
             'core/tests/protractor_desktop/embedding.js',
+            'core/tests/protractor/accessibility.js'
```

Comment:
I don't think I can approve an addition here. But you don't seem to use it in any case?